update to use non deprecated pluginlib macro

mikaelarguedas opened the pull request:
These macros deprecated for now 8 years will be removed in the next ROS release (ROS Melodic)

Close #167

mikaelarguedas committed:
update to use non deprecated pluginlib macro

paulbovbel reviewed (approved):
LGTM, thanks @mikaelarguedas!

wkentaro committed:
Drop yakkety from CI: Yakkety went EOL in July

Close https://github.com/ros-perception/perception_pcl/issues/166

https://discourse.ros.org/t/suspension-of-debian-packaging-for-eol-ubuntu-yakkety/2444
